src/app/config.rs

```diff
@@ -299,8 +299,8 @@ const DEFAULT_RESOLV_CONF: &str = "/etc/resolv.conf";
 
 /// It's assumed that a typical proxy can serve inbound traffic for up to 100 pod-local
 /// HTTP services and may communicate with up to 10K external HTTP domains.
-const DEFAULT_INBOUND_ROUTER_CAPACITY: usize = 100;
-const DEFAULT_OUTBOUND_ROUTER_CAPACITY: usize = 100;
+const DEFAULT_INBOUND_ROUTER_CAPACITY: usize = 10000;
```

Comment:
Just out of curiosity, how did we pick this number?

Reply:
After talking with @olix0r about it, it is an arbitrary number that seemed more sensible than 100.

Reply:
Yeah, let's keep inbound at 100 for now. I can't think of a good reason for it to be so large...

Reply:
Sounds good.